docs/3.0.0-beta.x/community/contribute-with-docs.md

```diff
@@ -0,0 +1,486 @@
+# Contribute with Docs
+
+All efforts to contribute to the [Docs](https://strapi.io/documentation/3.0.0-beta.x/) are highly appreciated, we recommend [these guidelines](/3.0.0-beta.x/community/contribute-with-docs.html#the-workflow) **and** that you talk to a maintainer before spending much time making a pull request that may not align with the project roadmap.
+
+You are welcome to create tutorials, articles, and videos for your blog or channels without speaking to a maintainer.
```

Comment:
@soupette **"You are welcome to create tutorials, articles, and videos for your blog or channels without speaking to a maintainer."** was added as I have been asked if they were "allowed" to make tutorials on their blogs or youtube channels.

Reply:
I am thinking that this should go in a complete different section...

Also, I am not sure the maintainers (including me) want to be asked for permissions before contributing to the codebase as the community is growing I have the feeling that the maintainers will be disturbed a lot, thus causing us not to respond to the contributors which won't leave a good image...

Of course, contributors can always come to us for asking guidelines, development tips or clarifications but I don't think that clearly stating that code contribution should be talked with maintainers before development is a good idea especially since we don't provide them a way to reach us. I also have the feeling that we might be asked for support which we currently don't do...

Reply:
@soupette  - I basically copied what was there before and perhaps cleaned up the English, a little bit. Should I just delete the asking for permissions part? It's a little awkward, I agree.

Reply:
Yes I think it's better to delete it completely

Reply:
@soupette - I deleted this section.

Reply:
Great